### ISSUE TITLE
longer text

### ISSUE THREAD

**Issue by cbasavaraj** [BelBES/crnn-pytorch#9]
Hi, 
Thanks for the repo. It's very well coded and easy to use with a custom dataset.
I first tried on a custom dataset where the average text length is 7 letters. This works quite well.
Now using a more complicated dataset with average text length of 18 characters, where space can be one of the characters (so multiple words instead of single words). Still training (I think the GRU takes time), but so far the results have not been that good.
With both models, the loss goes down quite smoothly, but the average edit distance jumps around quite  a lot. For the first model, it improved when I used Adam (your default) instead of AdaDelta which I was playing with. For the second, Adam's not really doing the trick.
If you've worked a lot with this model and have some ideas, please let me know. Thanks.

**Comment by cbasavaraj** [comment]
For anyone else interested in this, it helps to have a bigger projection layer. I changed the default of 10x20 to 10x50, immediately got a jump. Plus using ResNet50 instead of ResNet18 also helps.

**Comment by BelBES** [comment]
Hi, I think that you should find an information bottlenecks in network. I didn't test this network with long text sequences.
